File: ios/RCTMGL-v10/RCTMGLFillLayerManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(RCTMGLFillLayerManager, RCTViewManager)

// circle layer props
RCT_EXPORT_VIEW_PROPERTY(sourceLayerID, NSString)

// standard layer props
#include "CommonLayerProperties.H"

@end

File: ios/RCTMGL-v10/RCTMGLFillLayerManager.swift
@objc(RCTMGLFillLayerManager)
class RCTMGLFillLayerManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
  
    override func view() -> UIView! {
      let layer = RCTMGLFillLayer()
      layer.bridge = self.bridge
      return layer
    }
}
